### PR TITLE
internal/v5: use entitycache for id handlers

### PR DESCRIPTION
**Pull request by rogpeppe**
A few less round trips...

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1451/
Test PASSed.

File: internal/v5/api.go [hunk 1]
```diff
@@ -1425,10 +1425,15 @@ func (h *ReqHandler) AuthIdHandler(f ResolvedIdHandler) ResolvedIdHandler {
 // ResolvedIdHandler returns an id handler that uses h.Router.Context.ResolveURL
 // to resolves any entity ids before calling f with the resolved id.
 //
+// The given fields will be added to the fields required by the cache, so
```
Comment (mhilton) [inline internal/v5/api.go]: s/The given/Any specified/ ?

**Comment by mhilton** [comment]
:+1:

File: internal/v5/api.go (hunk 1 above)
Comment (rogpeppe) [inline internal/v5/api.go]: done

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1452/
Test PASSed.

**Comment by fabricematrat** [comment]
:+1:

**Comment by rogpeppe** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**